mealpy.py

```diff
@@ -132,47 +133,50 @@ def cancel_current_meal(self):
 def get_mealpal_credentials():
     config = load_config()
     email = config['email_address']
-    if config['use_keyring']:
-        password = (
-            keyring.get_password(KEYRING_SERVICENAME, email)
-            or getpass.getpass('Credential not yet stored in keychain, please enter password: ')
-        )
-        keyring.set_password(KEYRING_SERVICENAME, email, password)
-    else:
-        password = getpass.getpass('Enter password: ')
+    password = getpass.getpass('Enter password: ')
     return email, password
 
 
+def initialize_mealpal():
+    root_dir = path.abspath(path.dirname(__file__))
+    cookies_path = path.join(root_dir, COOKIES_FILENAME)
+    mealpal = MealPal()
+
+    if path.isfile(cookies_path):
+        with open(cookies_path, 'rb') as cookies_file:
+            mealpal.cookies = pickle.load(cookies_file)
+
+        # hacky way of validating cookies
+        try:
+            mealpal.get_schedules('San Francisco')
```

Comment:
This is kind of hacky, but a simple way to check the validity of the cookies that have been loaded since this API requires valid cookies.